src/main/java/org/osiam/OsiamHome.java

```diff
@@ -33,11 +33,7 @@
 import org.springframework.stereotype.Component;
 
 import java.io.IOException;
-import java.nio.file.DirectoryStream;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
-import java.nio.file.StandardCopyOption;
+import java.nio.file.*;
```

Comment:
I couldn't care less than about imports. But whatever ... I'll fix it. Give me one moment, please.